packages/ui/src/uiManager.ts

```diff
@@ -22,7 +24,11 @@ export class UIManager {
         return playerContribRegistry.register(ResourceToken, 1, creator);
     }
 
-    constructor(private _options: UIManagerOptions) {}
+    constructor(private _options: UIManagerOptions) {
+        //todo [sa] call loadFont in a better way...
+        let fontManager = FontManager.getInstance();
+        fontManager.loadFont(this._options.corePlayer.config.contrib.ui.fonts);
```

Comment:
this will probably throw error if user didn't provide contrib configuration as it is handled before you merge the defaults. you can instead provide the corePlayer to the fontManager. Actually the font manager should be initialized by the ui manager as other managers are created so it will be easier for your to provide the core player :)